src/engraving/playback/renderers/noterenderer.cpp

```diff
@@ -93,7 +93,20 @@ void NoteRenderer::render(const Note* note, const RenderingContext& ctx, mpe::Pl
         return;
     }
 
-    result.emplace_back(buildNoteEvent(std::move(noteCtx)));
+    mpe::NoteEvent ev = buildNoteEvent(std::move(noteCtx));
+
+    if (ev.arrangementCtx().actualTimestamp > 0) {
```

Comment:
Previous fix was `>= 0` instead of `> 0`; I think `>=` would indeed make more sense, wouldn't it?